### DISCUSSION TITLE
Is there a way to redirect a netlify subdomain to a vercel domain?

### DISCUSSION THREAD

**Discussion by tettoffensive** [vercel/vercel#5116] [Help]
I have a jamstack site hosted on netlify. www.mydomain.com. I'd like to point admin.mydomain.com on netlify to my vercel site. Is there a way to do this? In the past I've created an `A` record with the subdomain pointing to an IP address for an AWS hosted site. But I imagine, the IP address isn't static.

**Comment by mcsdevv** [comment]
Hello, you can apply this advice in reverse: https://vercel.com/knowledge/pointing-subdomains-to-external-services

You may want to add the domain to Vercel first so that it generates the correct records for you to add.
